### ISSUE TITLE
Installing Plugin doesn't complete on version 4.0.3.16

### ISSUE THREAD

**Issue by joshguptill** [tomecode/show-me-password-sqldev-jdev#11]
When installing version 1.0.0 of your plugin into SQL Developer 4.0.3.16, the install indicates the plugin will finish installing after a restart.

After restarting, the plugin isn't available in the menu.  It acts like it didn't finish.

**Comment by tomecode** [comment]
Did you try to remove folder: `system_cache` from the application directory: `/SQL Developer/system*/system_cache` e.g.: in windows: `C:\Users\<user>\AppData\Roaming\SQL Developer\system<versonNumber>\system_cache`?

**Comment by joshguptill** [comment]
Yes, I tried that a few different times and ways and it didn't help.  I stopped SQL Developer from restarting so that I could manually close the app, delete the system_cache folder, and then start SD.  Also I tried just letting SD restart on its own (plug-in still not in menu), then I closed SD, then deleted the system_cache folder, then restarted SD.  Still no plug in visible.

**Comment by tomecode** [comment]
You can try a new version dedicated for SQLDeveloper versio from 4 to 17 [ShowMePasswordSQLDeveloper4-17_v1.1.0](https://github.com/tomecode/show-me-password-sqldev-jdev/releases/download/untagged-81d1eb54ae8d9dff2233/ShowMePasswordSQLDeveloper4-17_v1.1.0.zip)

**Comment by hemanthudimella** [comment]
Same is the case for SQL Developer 4.1.3.20. I tried your new version (ShowMePasswordSQLDeveloper4-17_v1.1.0) as well.
